hudi-sync/pom.xml

```diff
@@ -35,5 +35,6 @@
     <module>hudi-dla-sync</module>
     <module>hudi-hive-sync</module>
     <module>hudi-sync-common</module>
+    <module>hudi-bigquery-sync</module>
```

Comment:
Thinking out aloud, what you are saying is vendor code (like BigQuery) is in separate module but oss projects (like DataHub) can stay under hudi-sync? That makes sense. We can also add lock providers there for Spanner etc. @vingov You okay to move / rename the module? 

Orthogonally, do we need to add bundles for hudi - aws and hudi-gcloud ?

Reply:
I initially thought about it, then decided to go with bigquery name because, even without this bigquery sync, users can use hudi with gcp file system, I don't want to cause any confusion for the users, hence went ahead with bigquery and not gcp.

Based on @vinothchandar's suggestion, if we plan to add more gcp eco-system tools in this module, then would it make sense to keep it under hudi-sync module or does it demand a higher level module say hudi-gcp?

I'm okay to rename it after we decide on the correct strategy for future-proofing this module.

Reply:
@vingov a top level module like `hudi-gcp` makes sense, then we'll provide a gcp bundle for everything on gcp. We already have the top-level `hudi-aws`, in which we have glue sync tool. The structure makes things cleaner and easy for bundling and usage.

Reply:
That makes sense to me, I'll make the necessary changes.